### PR TITLE
Added ampSlotIndex to the whitelist of default allowed values.

### LINKED ISSUE #6063: [amp-ad] Uncaught Error: Unknown attribute for medianet: ampSlotIndex
This error occurs in the current canary for `amp-ad` of some specific ad networks (e.g., a9). It's introduced in #5837.

This error is harmless though, it will not affect ad rendering.

### PR DESCRIPTION
**Pull request by glevitzky**
Fixes #6063

### CODE REVIEW

File: 3p/3p.js [hunk 1]
```diff
@@ -271,6 +271,7 @@ function validateAllowedFields(data, allowedFields) {
     mode: true,
     consentNotificationId: true,
     container: true,
```
Comment (jridgewell) [inline 3p/3p.js]: Can you remove this one, it's not needed.

File: 3p/3p.js [hunk 2]
```diff
@@ -271,6 +271,7 @@ function validateAllowedFields(data, allowedFields) {
...
+    ampSlotIndex: true,
```
Comment (jridgewell) [inline 3p/3p.js]: Fine for now, but we need a better solution.

Comment (glevitzky) [inline 3p/3p.js]: I understand your apprehension, but this whole lifecycle reporter thing is temporary until we make use of amp analytics.

**Pull request merged**